server/models/updates/2015-01-05-mod-02.sql

```diff
@@ -0,0 +1,14 @@
+-- Updates to patient data
```

Comment:
@mbayopanda , these were removed in a separate pull request.  Please remove them in a separate commit.